src/gui_common/RadialMenu.cs

```diff
@@ -332,7 +334,6 @@ private void UpdateIndicator()
         }
 
         var center = Size / 2;
-        var indicatorOffset = center - new Vector2(IndicatorSize / 2.0f, IndicatorSize);
```

Comment:
What's the reasoning behind modifying this math?

Reply:
Honestly, I don't know. It just worked. Probably Godot update caused indicator center to be already centered without need to set offset or something like that.

Reply:
If having this math doesn't hurt, I'd prefer to keep it to make sure it works in the future as well (and just doesn't luckily end up with the correct position).

Reply:
Well, it did hurt so...

Reply:
Alright then, I'll check this without that code being in.

Reply:
It doesn't seem like the indicator lines up correctly...